inst/qml/jaspPlotBuilder.qml

```diff
@@ -0,0 +1,2932 @@
+            }
+
+            Group {
+                VariablesForm {
+
+                    removeInvisibles:	true
+                    preferredWidth: jaspForm.width - 2 * jaspTheme.contentMargin
+                    preferredHeight: 500
+
+                    infoLabel: qsTr("Input")
+                    AvailableVariablesList { name: "allVariablesList"}
+
+
+                    AssignedVariablesList {
+                        name: "variableRepeatedMeasures"
+                        title: "Repeated measures variables"
+                        visible: yesRM.checked
+                        property bool active: yesRM.checked
+                        onActiveChanged: if (!active && count > 0) itemDoubleClicked(0)
+                    }
+
+                    TextField {
+                        name: "rmFactorText"
+                        id: rmFactorText
+                        placeholderText: "Repeated-measures factor (e.g., Time, Dose, Concentration)"
+                        visible: yesRM.checked
+                        property bool active: yesRM.checked
+                        onActiveChanged: if (!active && count > 0) itemDoubleClicked(0)
+                        info: qsTr("Repeated-measures factor (e.g., Time, Dose, Concentration)")
+                        fieldWidth: 239
+                    }
+
+                    TextField {
+                        name: "dimensionText"
+                        id: dimensionText
+                        placeholderText: qsTr("Measurement dimension (e.g., cm, mg, minutes)")
+                        visible: yesRM.checked
+                        property bool active: yesRM.checked
+                        onActiveChanged: if (!active && count > 0) itemDoubleClicked(0)
+                        info: qsTr("Measurement dimension (e.g., cm, mg, minutes)")
+                        fieldWidth: 239
+                    }
+
+                    CheckBox {
+                    name:"useRMFactorAsFill"
+                    id:useRMFactorAsFill
+                    label: "Use RM factor as a group/color variable"
+                    visible: yesRM.checked
+                    }
+
+                    AssignedVariablesList {
+                        name: "variableYPlotBuilder"
+                        title: "Y axis"
+                        allowedColumns: ["scale", "ordinal", "nominal"]
+                        id: variableYPlotBuilder
+                        singleVariable: true
+                        visible: noRM.checked
+                        property bool active: noRM.checked
+                        onActiveChanged: if (!active && count > 0) itemDoubleClicked(0)
+                        info: qsTr("Select the variable for the Y-axis.")
+                        // onCountChanged: {
+                        //     if (count === 0 && addBoxplot.checked) {
+                        //         addBoxplot.checked = false;
+                        //     }
+                        // }
```

Comment:
```suggestion
```
please delete unused comments, in particular those of code, or explain (in a comment above the code) why it's there.